atr/form.py

```diff
@@ -69,6 +70,51 @@ class Widget(enum.Enum):
     URL = "url"
 
 
+class DistributionPlatformWrapper(enum.Enum):
```

Comment:
@sbp just realized this enum and mapping should probably go in `shared/distribution` if you agree